### ISSUE TITLE
Vegas 20 not supported

### ISSUE THREAD

**Issue by mediacloudusr** [satishsampath/frame-server#58]
Debugmode FrameServer does not work with Magix Vegas 20.
I have to manually install it in Vegas 20, and then it crashes Vegas 20 after creating the AVI file.

**Comment by satishsampath** [comment]
This frameserver hasn't been maintained for a while. Have you tried the other one? https://www.cumincode.com/frameserver/

**Comment by couleurm** [comment]
> This frameserver hasn't been maintained for a while. Have you tried the other one? [cumincode.com/frameserver](https://www.cumincode.com/frameserver/)

What happened to cumincode, it now redirects to your site

**Comment by satishsampath** [comment]
They tried a paid version, but had very few people buying it, so they gave it back to me and open sourced everything. You can get the latest for free at debugmode.com/frameserver/

**Comment by mediacloudusr** [comment]
Yes, I tried Cumincode. It works better with Vegas 20 (installation works for example) but things are getting crazy when I try to render two AVI files from two different Vegas 20 instances. Problems with frameserver windows and focus.... Terrible.
You should try by yourself using the free trial. https://www.vegascreativesoftware.com/us/vegas-pro/
